agent/engine/engine_unix_integ_test.go

```diff
@@ -779,6 +779,27 @@ func TestDockerStopTimeout(t *testing.T) {
 	}
 }
 
+func TestContainerStartTimeout(t *testing.T) {
```

Comment:
I don't think this test is necessary since this is already tested in `config_test.go`.

Reply:
OK, I will remove this test then.